### DISCUSSION TITLE
Possible potential for custom tags: uklotzde/aoide-rs

### DISCUSSION THREAD

**Discussion by sandersantema** [beetbox/beets#3987] [Ideas]
Have any of you seen this library? https://gitlab.com/uklotzde/aoide-rs

It seems like it would make arbitrary custom tags possible. 

Might be of interest for beets.

**Comment by wisp3rwind** [comment]
I don't quite see how this would help beets: We can already store arbitrary tags in the database ("flexible attributes/flexattrs"), the tricky bit is populating them with sensible data (where everybody has different expectations) and mapping them to the various tag formats when writing to media files. Going through a REST api and importing a Rust dependency will also complicate beets architecture as well as substantially overcomplicate installation.

**Reply by sandersantema** [reply]
Yeah now that I think of it you're right, been exploring a bit too much music tools lately. Beets is not the one which hasn't got custom tags :)
